core/google/cloud/_helpers.py

```diff
@@ -379,6 +379,29 @@ def _bytes_to_unicode(value):
         raise ValueError('%r could not be converted to unicode' % (value,))
 
 
+def _from_any_pb(pb_type, any_pb):
```

Comment:
[`google.cloud.operation._from_any`][1] already exists, can you "unify"?

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/59fd1e4ff50827a868986deee938a26b5d50dc0f/core/google/cloud/operation.py#L64-L81